sources/packages/backend/apps/api/src/utilities/class-validation/custom-validators/custom-validation-pipe.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  ArgumentMetadata,
+  Injectable,
+  ParseEnumPipe,
+  ParseEnumPipeOptions,
+} from "@nestjs/common";
+
+/**
+ * Custom validation pipe extended from ParseEnumPipe(Nest JS)
+ * to allow nullable values and validates the only if
```

Comment:
Is seems that "the" is not required in the middle of the sentence "validates the only".